python/lsst/analysis/drp/colorColorPlot.py

```diff
@@ -191,19 +209,22 @@ def colorColorPlot(self, catPlot, plotInfo):
         starCbAx = fig.add_axes([0.89, 0.11, 0.04, 0.75])
         plt.colorbar(starPoints, cax=starCbAx, extend="both")
         magLabel = self.config.axisLabels["z"]
-        galCbAx.text(0.6, 0.5, magLabel + ": Galaxies", color="k", rotation="vertical",
-                     transform=galCbAx.transAxes, ha="center", va="center", fontsize=10)
-        starCbAx.text(0.6, 0.5, magLabel + ": Stars", color="k", rotation="vertical",
-                      transform=starCbAx.transAxes, ha="center", va="center", fontsize=10)
+        galText = galCbAx.text(0.6, 0.5, magLabel + ": Galaxies", color="k", rotation="vertical",
```

Comment:
This is why all the colour bar axis labels are slightly lower than they should be. Every instance of this should be changed to:```
.text(0.5, 0.5
```